docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
+
+### What should I do if my Icebreaker node fails to register using IPv6?
+
+If registration fails using IPv6, consider the following:
+
+1. Ensure your IPv6 address is reachable by running:
+   ```bash
+   curl -v http://your-ipv6-address:port/
+   ```
+2. Temporarily disable IPv6 or use the `-4` CLI switch to force IPv4 usage.
+3. Refer to the [GitHub issue #207](https://github.com/blockfrost/blockfrost-platform/issues/207) for updates on IPv6 support.
+
+### How do I handle IPv6 connectivity issues?
+
+For IPv6 issues:
+
+1. Verify that your IPv6 address is publicly reachable.
+2. Use the `-4` CLI switch to force IPv4 if necessary.
+
+### I see warnings about old UUID paths after restarting my Icebreaker. Should I be concerned?
+
+No, these warnings are typically not a concern.
+Each restart generates a new random UUID path, and the Blockfrost servers may briefly attempt to contact the old path.
+Unless the warnings persist, they can be safely ignored.
+This is expected behavior and should not cause concern unless persistent issues occur.
+If so, check your logs for errors.
+
+## Maintenance and advanced topics
+
+### What is the process for updating my Icebreaker node to a new version of the Blockfrost platform?
+
+When a new version of the Blockfrost platform is released:
+
+1. Watch for an announcement in the private Icebreakers Discord channel with update instructions
+2. Shut down your current `blockfrost-platform` process
+3. Update the platform software using the same [installation method](installation/installers) you originally used
+4. Restart the `blockfrost-platform` process
+5. Verify the new version is running with `blockfrost-platform --version`
+6. Monitor your logs and the dashboard to ensure your node reconnects successfully.
+
+Promptly updating is recommended to maintain compatibility and access new features.
+The specific update steps may vary slightly between releases so always refer to the official release notes.
+
+### What are the rules around sharing my Icebreaker secret key?
+
+Your Icebreaker secret key is like a password for your node.
+It should be kept confidential and never shared publicly.
+The secret key authorizes your node to participate in the Icebreakers platform.
+
+Specific guidelines:
+
+1. Don't post your secret key on any public websites, forums, or social media
+2. Don't share the key with untrusted parties
+3. Only store the key on servers you control and trust
+4. Use a unique secret key for each Icebreaker node you run
+5. If you suspect your key is compromised, generate a new one and update your node.
+
+The Blockfrost team will never ask for your secret key.
+Be cautious of phishing attempts.
+
+If you lose your key, reach out to the team on Discord for assistance with generating a new one.
+
+### How can I improve the security of my Icebreaker node?
+
+For enhanced security:
+
+1. Limit access to the Blockfrost platform to specific IP addresses.
```

Comment:
Although we are suggesting this, this can't be achieved in our current setup. After WebSockets connection is in production, they can easily secure it.

Reply:
@ginnun Are you referring to this same issue: 

"We have implemented a reverse-connection mechanism using WebSockets. While this feature is not yet in production, it is designed to simplify node operation in complex network environments, such as those involving firewalls, double NAT, CGNAT, or the absence of a static IP address."

If so, I can weave that message in here.

Reply:
@joseph-fajen Yes that's it.